riscv/riscv.mk.in

```diff
@@ -1361,22 +1358,16 @@ riscv_insn_list = \
 	$(riscv_insn_svinval) \
 	$(riscv_insn_ext_cmo) \
 
-riscv_fast_gen_srcs = $(addsuffix _fast.cc,$(riscv_insn_list))
-riscv_logged_gen_srcs = $(addsuffix _logged.cc,$(riscv_insn_list))
-
-riscv_gen_srcs = $(riscv_fast_gen_srcs) $(riscv_logged_gen_srcs)
+riscv_gen_srcs = $(addsuffix .cc,$(riscv_insn_list))
 
 insn_list.h: $(src_dir)/riscv/riscv.mk.in
 	for insn in $(foreach insn,$(riscv_insn_list),$(subst .,_,$(insn))) ; do \
 		printf 'DEFINE_INSN(%s)\n' "$${insn}" ; \
 	done > $@.tmp
 	mv $@.tmp $@
 
-$(riscv_fast_gen_srcs): %_fast.cc: insns/%.h insn_template.cc
-	sed 's/NAME/$(subst _fast.cc,,$@)/' $(src_dir)/riscv/insn_template.cc | sed 's/TYPE/fast/' | sed 's/OPCODE/$(call get_opcode,$(src_dir)/riscv/encoding.h,$(subst _fast.cc,,$@))/' > $@
-
-$(riscv_logged_gen_srcs): %_logged.cc: insns/%.h insn_template.cc
-	sed 's/NAME/$(subst _logged.cc,,$@)/' $(src_dir)/riscv/insn_template.cc | sed 's/TYPE/logged/' | sed 's/OPCODE/$(call get_opcode,$(src_dir)/riscv/encoding.h,$(subst _logged.cc,,$@))/' > $@
+$(riscv_gen_srcs): %.cc: insns/%.h insn_template.cc
+	sed 's/NAME/$(subst .cc,,$@)/' $(src_dir)/riscv/insn_template.cc | sed 's/TYPE/fast/' | sed 's/OPCODE/$(call get_opcode,$(src_dir)/riscv/encoding.h,$(subst .cc,,$@))/' > $@
```

Comment:
Extra `sed 's/TYPE/fast/` should be removed

Reply:
oops!